packages/documentation/src/pages/architecture/unit-tests.mdx

```diff
@@ -0,0 +1,120 @@
+## How do unit tests work?
+
+#### Where they live
+- Tests can live anywhere in the `src` directory, but in general are located in a directory close to the code or application that they are written to test.
+- The `.unit.spec.js` file extension is used for unit tests, while `.e2e.spec.js` is used for end-to-end (E2E) tests.
+- Visual regression test config has its own directory under `src/platform/testing/visual-regression`
```

Comment:
I think there's another document describing the VRT so we can probably just reference that document or remove this line.